great_expectations/execution_engine/sparkdf_execution_engine.py

```diff
@@ -301,6 +302,29 @@ def get_batch_data_and_markers(
                     f"""Unable to read in batch from the following path: {path}. Please check your configuration."""
                 )
 
+        elif isinstance(batch_spec, GlueCatalogBatchSpec):
```

Comment:
@lccasagrande If the handling of the `GlueCatalogBatchSpec` is identical to that of another `BatchSpec` subclass, might we be able to combine them by using a `tuple` in the appropriate use of `isinstance()`?  Thanks!

Reply:
Fixed